src/extension.ts

```diff
@@ -39,6 +39,17 @@ export function activate(context: vscode.ExtensionContext) {
         client.onServerOutputAppended(event => {
             serversData.addServerOutput(event);
         });
+        client.onStringPrompt((event, resolve, reject) => {
+            vscode.window.showInputBox({prompt: event.prompt, password: true}).then(value => {
+                if (value && value.length) {
```

Comment:
Current condition would not work for `"     "` should be `value && value.trim()`. Have not tested it, just my assumption based on truthy/falsy definition.